modules/core/arrow-test/src/main/kotlin/arrow/test/laws/ConcurrentLaws.kt

```diff
@@ -98,25 +98,26 @@ object ConcurrentLaws {
       Law("Concurrent Laws: onError outer and inner finalizer is run when error is raised") { CF.waitForTimesOutProgram(EQ) },
       Law("Concurrent Laws: onError outer and inner finalizer is run when error is raised") { CF.waitForTimesOutProgramWithDefault(EQ) }
     ) + (if (testStackSafety) {
+      val iterations = 20_000
```

Comment:
The reason why we had 5k was not to make the machine OOM, and keep the runtimes reasonable.

Reply:
@pakoito alright, shall we make it configurable and default to `5_000`? It's currently also still fixed to `20_000` in `MonadDeferLaws`.

Reply:
Worth a try to keep it at 20k then. Configurable, for sure!